### ISSUE TITLE
Switching to a11y mode broke BlueImp galllery

### ISSUE THREAD

**Issue by roman-yagodin** [roman-yagodin/R7.Epsilon#30]
Switching to a11y mode broke BlueImp galllery, possibly due to a11y parameter in the URL.

**Comment by roman-yagodin** [comment]
The issue shows up when images passed to the BlueImp gallery by relative paths starting with `/`, so when DNN Advanced URLs rewrite `?a11y=true[false]` to `/a11y/true[false]` the resources cannot be loaded by the browser. Solution is to use relative paths w/o starting `/`.
